front/ticket.form.php

```diff
@@ -281,7 +281,7 @@
         Html::header(sprintf(__('%s Kanban'), Ticket::getTypeName(1)), '', "helpdesk", "ticket");
         $track::showKanban(0);
         Html::footer();
-    } else {
+    } else if (isset($_GET['id'])) {
```

Comment:
There is a `unset($_GET['id']);` on line 263. So, it will never reach this piece of code.

Reply:
In fact we are on the else scope of the condition `if (isset($_GET["id"]) && ($_GET["id"] > 0))` line 227. The id can't exist there. I think we can safely remove the else completely.